biodatasets/quaero/quaero.py

```diff
@@ -0,0 +1,280 @@
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="quaero_source",
+            version=SOURCE_VERSION,
+            description="QUAERO source schema",
+            schema="source",
+            subset_id="quaero",
+        ),
+        BigBioConfig(
+            name="quaero_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="QUAERO simplified BigBio schema",
+            schema="bigbio_kb",
+            subset_id="quaero",
+        ),
+    ]
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "entities": [
+			            {
+			                "id": datasets.Value("string"),
+			                "type": datasets.Value("string"),
+			                "text": datasets.Sequence(datasets.Value("string")),
+			                "offsets": datasets.Sequence([datasets.Value("int32")]),
+			                "normalized": [
+			                    {
+			                        "db_name": datasets.Value("string"),
+			                        "db_id": datasets.Value("string"),
+			                    }
+			                ],
+			            }
+			        ],
+			        "passages": [
+			            {
+			                "id": datasets.Value("string"),
+			                "type": datasets.Value("string"),
+			                "text": datasets.Sequence(datasets.Value("string")),
+			                "offsets": datasets.Sequence([datasets.Value("int32")]),
+			            }
+        ],
+                      }
+            )
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+            
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _get_bioc_entity(self, span, doc_text, db_id_key=None):
+        """Parse BioC entity annotation.
+        Parameters
+        ----------
+        span : BioCAnnotation
+            BioC entity annotation
+        doc_text : string
+            document text, required to construct text spans
+        db_id_key : str, optional
+            database name used for normalization, by default "MESH"
+        Returns
+        -------
+        dict
+            entity information
+        """
+        offsets = [(loc.offset, loc.offset + loc.length) for loc in span.locations]
+        texts = [doc_text[i:j] for i, j in offsets]
+        db_ids = span.infons[db_id_key] if db_id_key else "-1"
+        normalized = [
+            # some entities are linked to multiple normalized ids
+            {"db_name": db_id_key, "db_id": db_id}
+            for db_id in db_ids.split("|")
```

Comment:
Have you checked that this is the only `connnector`? They could have used `+`, `,` as well